src/plugins/hiddenRows/contextMenuItem/showRow.js

```diff
@@ -62,7 +62,7 @@ export default function showRowItem(hiddenRowsPlugin) {
         return this.toPhysicalRow(visualRowIndex);
       });
 
-      if (!this.selection.isSelectedByRowHeader() || hiddenPhysicalRows.length < 1) {
+      if ((!this.selection.isSelectedByRowHeader() && !this.selection.isSelectedByCorner()) || hiddenPhysicalRows.length < 1) {
```

Comment:
The line exceeds 120 characters. Additionally, I didn't found any test about visibility the "Show rows" option in the context menu when the corner is clicked. Can I ask you to fix that?

You updated the expression for the _HiddenRows_ plugin but the same should be applied for the _HiddenColumns_ plugin. If you don't want to mix threads maybe it's a better idea to finish this in separate PR (and restore all context menu items changes)?

Reply:
> You updated the expression for the HiddenRows plugin but the same should be applied for the HiddenColumns plugin.

True, I'll do that.

I don't think this issue is a good place to add tests for the hidden rows/hidden columns plugins, but I'm afraid that if I create a separate issue for it and throw it on the backlog pile, we'll never get to it, so I'll add them here as well.